low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -32,18 +31,100 @@ class ServerMap(ResourceMap):
 
     def __init__(self):
         """Initialize server"""
-        pass
+        super().__init__()
+        ipmi_factory = IpmiFactory()
```

Comment:
Do this ipmi specific init in separate function

Reply:
We need this `self._ipmi` through out the module. I have merged the initialization into one line.